orttraining/orttraining/core/optimizer/memory_optimizer.h

```diff
@@ -73,20 +72,22 @@ class MemoryAlleviation : public GraphTransformer {
   };
 
   /**
-   * @brief A struct to maintain the information of recomputable subgraphs.
-   * Imagine we loop all nodes finding recomputable subgraphs, we want to store them first.
-   * Afterwards, we optionally pick up some of them to apply alleviation according to user configs.
+   * @brief A struct to maintain the information of target subgraphs to optimize.
+   * Imagine we loop all nodes finding recomputable/offload-able subgraphs, we want to store them first.
+   * Afterwards, we optionally pick up some of them to apply optimization according to user configs.
    *
    * subgraph_descs is a map from subgraph string representation to its subgraph related configurations.
-   * recompute_graphs is a map from activation producer node pointers to its recomputable subgraph nodes.
+   * _optimization_target_graphs_ is a map from activation producer node pointers to its target optimization subgraph
```

Comment:
"activation producer node pointers" is this the node which produces the activation?
 what is "target optimization subgraph nodes" ?

Reply:
""activation producer node pointers" is this the node which produces the activation?" yes it is. 
 "target optimization subgraph nodes" is the graphs that is going to be recomputed or offloaded.

Reply:
I added a sample to illustrate it.